doc/help/embed.md

```diff
@@ -0,0 +1,271 @@
+# Introduction
+
+# Login
+
+![Login](login.png)
+
+# User Management
```

Comment:
As a regular user (e.g. using fran+demo login) I don't see this tab. Clarify that only super users have it?